AK/Traits.h

```diff
@@ -41,6 +42,11 @@ requires(IsPointer<T>) struct Traits<T> : public GenericTraits<T> {
     static constexpr bool is_trivial() { return true; }
 };
 
+template<Enum T>
+struct Traits<T> : public GenericTraits<T> {
+    static unsigned hash(T value) { return Traits<UnderlyingType<T>>::hash(to_underlying(value)); }
```

Comment:
Please also make `is_trivial()` return `true` (or proxy it to the underlying type trait)